test/Dialect/FIRRTL/lower-types.mlir

```diff
@@ -1094,56 +1089,54 @@ firrtl.circuit "NLALowering" {
-    %bundle2 = firrtl.wire sym @b2 {
+
+    %bundle2 = firrtl.wire {
       annotations = [
         {circt.fieldID = 3 : i32, circt.nonlocal = @nla_b2, class = "firrtl.transforms.DontTouchAnnotation"},
         {circt.fieldID = 2 : i32, circt.nonlocal = @nla_b2, class = "firrtl.transforms.DontTouchAnnotation"}
       ]
     } : !firrtl.bundle<valid: uint<1>, ready: uint<1>, data: uint<64>>
-    // CHECK:   %bundle_valid = firrtl.wire sym @b_valid : !firrtl.uint<1>
-    // Note that same NLA is reused. But this depends on the order of the annotations, if DontTouch was earlier in the list, it would be dropped and a new NLA would be created.
-    // CHECK-NEXT:   %bundle_ready = firrtl.wire sym @b_ready {annotations = [{circt.nonlocal = @nla_b_0, class = "test"}]} : !firrtl.uint<1>
+    // CHECK:   %bundle_valid = firrtl.wire : !firrtl.uint<1>
+    // CHECK-NEXT:   %bundle_ready = firrtl.wire
+    // CHECK-SAME:   {annotations = [{circt.nonlocal = @nla_b, class = "test"}, {circt.nonlocal = @nla_b, class = "firrtl.transforms.DontTouchAnnotation"}]} : !firrtl.uint<1>
     // Note the new NLA added here.
-    // CHECK-NEXT:   %bundle_data = firrtl.wire sym @b_data
-    // CHECK-SAME: {annotations = [{B, circt.nonlocal = @nla_b_1}, {A, circt.nonlocal = @nla_b_1}]}
-    // CHECK:   %bundle2_valid = firrtl.wire sym @b2_valid  : !firrtl.uint<1>
-    // CHECK:   %bundle2_ready = firrtl.wire sym @b2_ready  : !firrtl.uint<1>
-    // CHECK:   %bundle2_data = firrtl.wire sym @b2_data  : !firrtl.uint<64>
+    // CHECK-NEXT:   %bundle_data = firrtl.wire
+    // CHECK-SAME: {annotations = [{B, circt.nonlocal = @nla_b}, {A, circt.nonlocal = @nla_b}, {circt.nonlocal = @nla_b, class = "firrtl.transforms.DontTouchAnnotation"}]}
+    // CHECK:   %bundle2_valid = firrtl.wire : !firrtl.uint<1>
+    // CHECK:   %bundle2_ready = firrtl.wire {annotations = [{circt.nonlocal = @nla_b2, class = "firrtl.transforms.DontTouchAnnotation"}]}
+    // CHECK:   %bundle2_data = firrtl.wire {annotations = [{circt.nonlocal = @nla_b2, class = "firrtl.transforms.DontTouchAnnotation"}]}
   }
 }
 
 // Test the update of NLA when a new symbol is added after lowering of bundle fields.
 firrtl.circuit "NLALoweringNewSymbol" {
-  firrtl.hierpath @lowernla_2 [@NLALoweringNewSymbol::@testBundle_Bar, @testBundle_Bar::@d]
-  // CHECK: firrtl.hierpath @lowernla_2 [@NLALoweringNewSymbol::@testBundle_Bar, @testBundle_Bar::@d_baz]
-  // CHECK: firrtl.hierpath @lowernla_2_0 [@NLALoweringNewSymbol::@testBundle_Bar, @testBundle_Bar::@d_qux]
-  firrtl.hierpath @lowernla_1 [@NLALoweringNewSymbol::@testBundle_Bar, @testBundle_Bar::@b]
-  // CHECK: firrtl.hierpath @lowernla_1 [@NLALoweringNewSymbol::@testBundle_Bar, @testBundle_Bar::@b_baz]
-  // CHECK: firrtl.hierpath @lowernla_1_0 [@NLALoweringNewSymbol::@testBundle_Bar, @testBundle_Bar::@b_qux]
-  // CHECK: firrtl.hierpath @lowernla_1_1 [@NLALoweringNewSymbol::@testBundle_Bar, @testBundle_Bar::@b_data]
```

Comment:
I assumed hierpath with old format will never be created. Is it correct?

Reply:
It could be created by somebody, but not be a "normal" pass pipeline that uses `LowerAnnotations`. It's fine to remove this.